thunder/torch/default_torch_ops.py

```diff
@@ -693,3 +694,11 @@
         torch.special.xlogy,
     ],
 }
+
+
+def _check_is_auto_registered_torch_op(op: Callable) -> str | None:
+    for k, vals in torch_auto_registered_ops.items():
+        if op in vals:
+            op_name = op.__name__.split("_")[1] if k is torch.special else op.__name__
```

Comment:
Since this handling comes up in a few places, maybe there should be a common function for the name resolution?